content/posts/writing-posts/markdown-guide/markdown-syntax-guide.md

```diff
@@ -4,7 +4,7 @@ date: 2020-06-08T06:15:40+06:00
 hero: /images/posts/writing-posts/code.svg
 author:
   name: Md. Emruz Hossain
-#   image: /assets/images/profile-image.jpg
+  image: /assets/images/profile-image.jpg
```

Comment:
This image name is not longer valid. It has been renamed to `default-avatar.png`. 
https://github.com/hossainemruz/toha/tree/master/static/assets/images